db/migrate/20221010090227_add_devise_two_factor_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeviseTwoFactorToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :encrypted_otp_secret, :string
+    add_column :users, :encrypted_otp_secret_iv, :string
```

Comment:
what is `encrypted_otp_secret_iv` ?

Reply:
The version that we are currently using for devise-two-factor (4.0.2) generates these columns where encrypted_otp_secret holds to secret but I dont know the exact purpose of encrypted_otp_secret_iv. But these columns are removed in the latest version of devise-two-factor by a single column 'otp_secret'. Ref: https://github.com/tinfoil/devise-two-factor/blob/main/UPGRADING.md